doc/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,149 @@
+# ADR 1: Record architecture decisions
+
+Date: 2022-06-08
+
+## Status
+
+Draft
+
+## Context
+
+We are in search for a means to document our architectural and design decisions
+for all of our components.
+In order to do that, there is practice called architectural design records, ADR,
+that we can integrate into our workflow.
```

Comment:
If ADR is a strict abbreviation that you want to use throughout, I would introduce it like this: 

In order to do that, there is a practice called architectural design records ("ADR") that we will be integrating into our workflow.